dashboard/config/environments/adhoc.rb

```diff
@@ -6,4 +6,27 @@
 Dashboard::Application.configure do
   # Show full error reports
   config.consider_all_requests_local       = true
+
+  # Use digests.
```

Comment:
Might want to include this directive from the production environment too - it introduced some differences in local/production for me in recent work.

```ruby
  # Do not fallback to assets pipeline if a precompiled asset is missed.
  config.assets.compile = false
```